cli/src/main/java/org/jboss/as/cli/operation/impl/DefaultOperationCandidatesProvider.java

```diff
@@ -210,6 +212,193 @@ public CommandLineCompleter getCompleter() {
         return Util.getOperationNames(ctx, prefix);
     }
 
+    /**
+     * Class that establishes completion visibility of a property.
+     * If all (could be double TAB), hidden properties (due to alternatives/requires)
+     * are shown.
+     */
+    public static class PropertyVisibility {
+        private static final String REQUIRED_PROPERTY = "*";
+        private final Set<String> presentProperties;
+        private final String radical;
+        private final boolean all;
+        private final boolean multiple;
+        private final Set<String> invalidProperties = new HashSet<>();
+        private final List<Property> propList;
+        private final Map<String, Set<String>> requiredBy = new HashMap<>();
+        public PropertyVisibility(boolean all, List<Property> propList,
+                Set<String> presentProperties, String radical) {
+            this.all = all;
```

Comment:
I don't see this set to 'true' anywhere. Is that planned for the future?

Reply:
Good point, it was planed at some point but removed, I removed it.